src/TesApi.Web/BatchScheduler.cs

```diff
@@ -415,9 +415,11 @@ public void ClearBatchLogState()
         private static string GetCromwellExecutionDirectoryPath(TesTask task)
             => GetParentPath(task.Inputs?.FirstOrDefault(IsCromwellCommandScript)?.Path.TrimStart('/'));
 
-        private static string GetStorageUploadPath(TesTask task, string storageAccountName)
+        private string GetStorageUploadPath(TesTask task)
```

Comment:
Can you add a function descriptor comment? I'm able to follow because I just worked on this code/read the github issue but I'd be pretty confused as to why we're doing this if I was just reading the code.

Reply:
https://github.com/microsoft/ga4gh-tes/issues/253